lib/ldp.js

```diff
@@ -473,11 +479,12 @@ LDP.prototype.deleteResource = function (filename, callback) {
 
 LDP.prototype.getAvailablePath = function (host, containerURI, slug, callback) {
   var self = this
+  var exists
 
   if (!slug) {
     slug = uuid.v1()
   }
-  var exists
```

Comment:
this is used later on in the code (487 | 494), it can't be deleted or otherwise the other would become a global variable (and we don't want that)